### PR TITLE
VWO Device mode Identify call

### PR DESCRIPTION
**Pull request by prathameshwingify**
_A summary of your pull request, including the what change you're making and why._
Added additional settings fields,` identifyUser ` action added
## Testing

_Include any additional information about the testing you have completed to
ensure your changes behave as expected. For a speedy review, please check
any of the tasks you completed below during your testing._

- [x] Added [unit tests](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing) for new functionality
- [x] Tested end-to-end using the [local server](https://github.com/segmentio/action-destinations/blob/main/docs/testing.md#local-end-to-end-testing)
- [ ] [Segmenters] Tested in the staging environment

### CODE REVIEW

File: packages/browser-destinations/src/destinations/vwo/index.ts [hunk 1]
```diff
@@ -23,26 +25,60 @@ export const destination: BrowserDestinationDefinition<Settings, VWO> = {
...
     }
   ],
   settings: {
     // Add any Segment destination settings required here
     vwoAccountId: {
-      description: 'The VWO AccountID from VWO dashboard',
+      description: 'Your VWO account ID, used for fetching your VWO async smart code.',
       label: 'VWO Account ID',
       type: 'number',
+      default: 654331,
```
Comment (pooyaj) [inline packages/browser-destinations/src/destinations/vwo/index.ts:42]: Is this right? do we want to hardcode this?

File: packages/browser-destinations/src/destinations/vwo/index.ts [hunk 2]
```diff
@@ -23,26 +25,60 @@ export const destination: BrowserDestinationDefinition<Settings, VWO> = {
...
+      label: 'Settings Tolerance',
+      type: 'number',
+      default: 2000
+    },
+    libraryTolerance: {
+      description:
+        'The maximum amount of time (in milliseconds) to wait for VWO’s full library to be downloaded before simply displaying your original page.',
+      label: 'Library Tolerance',
+      type: 'number',
+      default: 2000
```
Comment (pooyaj) [inline packages/browser-destinations/src/destinations/vwo/index.ts:57]: Can we make this `default` value consistent with the hard-coded value in `init-script.ts` ( default parameter value in `initScript({ ...`  ( currently we have 2000 in one place and 2500 in the other )

**Review by pooyaj — COMMENTED** [review]
Two minor comments, otherwise looks good!

File: packages/browser-destinations/src/destinations/vwo/index.ts (hunk 1 above)
Comment (prathameshwingify) [reply packages/browser-destinations/src/destinations/vwo/index.ts]: It has been corrected

File: packages/browser-destinations/src/destinations/vwo/index.ts (hunk 2 above)
Comment (prathameshwingify) [reply packages/browser-destinations/src/destinations/vwo/index.ts]: It has been corrected.

**Review by pooyaj — APPROVED** [review]
Looks great 🎉

**Pull request merged**